src/planner/select_plan_test.go

```diff
@@ -290,7 +290,6 @@ func TestSelectUnsupportedPlan(t *testing.T) {
 		"select id, rand(id) from A",
```

Comment:
`select * from A as A where id in (select id from B)` need handle

Reply:
maybe more cases?

Reply:
OK, this line has being fixed.